pkg/testfixtures/storage/storage.go

```diff
@@ -52,3 +59,26 @@ func RunDatastoreTestContainer(t testing.TB, engine string) DatastoreTestContain
 		return nil
 	}
 }
+
+func MustBootstrapDatastore(t testing.TB, engine string) storage.OpenFGADatastore {
+	testDatastore := RunDatastoreTestContainer(t, engine)
+
+	uri := testDatastore.GetConnectionURI(true)
+
+	var ds storage.OpenFGADatastore
+	var err error
+
+	switch engine {
+	case "memory":
+		ds = memory.New(10, 24)
+	case "postgres":
+		ds, err = postgres.New(uri, sqlcommon.NewConfig())
+	case "mysql":
+		ds, err = mysql.New(uri, sqlcommon.NewConfig())
+	default:
+		t.Fatalf("'%s' is not a supported datastore engine", engine)
```

Comment:
Should we have BootstrapDatastore as part of datastore interface (or something like testdatastore interface)?  In this way, custom datastore does not need to modify this method.  Instead, we have compile time check rather than test/run-time check.

Reply:
It's a utility method only used in tests. I don't think it belongs on the interface, as it is not needed for any storage functionality other than to help bootstrap a datastore implementation in tests.

Reply:
If someone have another storage adaptor (i.e., dynamo for example), what do they need to do?  Do they update this file or override the functionality?  My concern here is that the textfixture/storage/storage.go is generic and should not have any knowledge of whether there is mysql/postgresql/memory etc.

Reply:
It's not generic though. It has to have knowledge about the supported datastore engines in order to provide a `DatastoreTestContainer` to callers. See https://github.com/openfga/openfga/blob/main/pkg/testfixtures/storage/storage.go#L42

In fact, we also must provide implementations of the `DatastoreTestContainer` for each implementing storage engine.

https://github.com/openfga/openfga/tree/main/pkg/testfixtures/storage

This package is just meant to help with construction of a test container for testing purposes. In any case, I moved this function out and into the place directly where it is used.